src/clj/game/core-io.clj

```diff
@@ -81,10 +81,15 @@
 
 
 ; In-game chat commands
```

Comment:
While you are editing this file, please throw a two more `;` here (so it starts with `;;; `). Makes it a nice header-like comment.